### ISSUE TITLE
[API v.2] WebApp to have more features from Desktop

### ISSUE THREAD

**Issue by maksimdzmitryew** [moneymanagerex/web-money-manager-ex#30]
Hello @vomikan

I am thinking of new API version. So we could plan adding new features to WebApp.

1. Do we (you?) have a martix of features in DesktopApp vs WebApp? 
2. Is it possible to have statistical information from WebApp version? So we could see when API v.1 goes into decline?

What I immediately see are these ones: 

1. WebApp does not have a "favorite" account option.
2. Another feature that I miss when "on the go" is the account's current balance availability in the WebApp. I understand it might go out of sync if I use DesktopApp in parallel with WebApp. But this a rear thing to happen until I'm back. And then I import data from WebApp. So discrepancy is likely not to be the case. We could maybe add a disclaimer "Balance might not reflect actual data blah-blah-blah", though.

**Comment by vomikan** [comment]
I read this message. There are no thoughts on this yet. One thing I want to note is that this interface is like importing accumulated transactions at the start of the main application. If the main application is not running, there can be no question of any data exchange.
The main thing is not to turn this into a web version of a desktop application.

**Comment by maksimdzmitryew** [comment]
I agree that WebApp is a complementary (additional) functionality to Desktop. I would not like to overload it with features either. I like the way it is now: simple, light-weight, and fast. However I miss some small "convenient" things that would make UX much better with a comparatively smaller effort.

I understand that account information is exchanged between Desktop and Web only when Desktop is up and running.

**Comment by vomikan** [comment]
I can support any feature. If something is required from the desktop application, just make a case in https://github.com/moneymanagerex/moneymanagerex/issues

I don't have my own ideas.

**Comment by luisdev** [comment]
Adding support for the new Tags feature in MMX 1.7 would be wonderful.
